src/pages/calendar/components/Section.tsx

```diff
@@ -59,9 +78,15 @@ export function SectionInfo({
   seat,
   meetingTimes,
 }: SectionInfoProps): JSX.Element {
-  const isASYNC = additionalNotes?.indexOf('asynchronous') !== -1;
+  const isASYNC = additionalNotes?.indexOf('This course will be offered fully online and asynchronous') !== -1;
+  const isBLENDED = additionalNotes?.indexOf('a mix of “real-time” and asynchronous sessions') !== -1;
   const isSENG = additionalNotes?.indexOf('Reserved for BSENG students') !== -1;
   const isCSC = additionalNotes?.indexOf('Reserved for students in a Computer Science program') !== -1;
+  const isENGR = additionalNotes?.indexOf('Restricted to BEng students only') !== -1;
+  const isENGRorCSC = additionalNotes?.indexOf('Restricted to students in the Faculty of Engineering') !== -1;
+  const isSCI = additionalNotes?.indexOf('Restricted to SCIENCE students') !== -1;
+  const isBENGorBSENG = additionalNotes?.indexOf('Restricted to BEng and BSEng students') !== -1;
+  const isENGRmulti = additionalNotes?.indexOf('Reserved for BME, BSEN, CENG, ELEC students') !== -1;
```

Comment:
Since this is getting a bit long maybe we can consider refactoring to something like the following:
```
let rules be an array of rules where a rule might look like "a mix of real-time"
let badges be an array of badges where a badge object contains a name and colour atrribute 
for each rule in rules
  if found add name and colour to badges array
end
```
then using a `badges` array, we could render them out like:
```jsx
{badges.map(badge => <CourseBadge name={name} color={colour}/>)
```